ansible/roles/stack-sunbird/templates/taxonomy-service_application.conf

```diff
@@ -390,3 +390,6 @@ languageCode {
   telugu : "te"
 }
 objectcategorydefinition.keyspace="{{ lp_cassandra_keyspace_prefix }}_category_store"
+
+# Framework master category validation
+master.category.validation.enabled="{{ master_category_validation_enabled }}"
```

Comment:
is there any default value for that?
if so please update
```
master.category.validation.enabled="{{ master_category_validation_enabled | default('true') }}" 
```

Reply:
Done @rjshrjndrn